### PR TITLE
add dev guide(zh)

### PR DESCRIPTION
**Pull request by shczhen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14705 https://github.com/pingcap/docs/pull/14532
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Icemap — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@shczhen PTAL https://github.com/shczhen/docs-cn/pull/1

File: develop/dev-guide-sample-application-aws-lambda.md
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-aws-lambda.md:1]: Preview: https://pingcap-docsite-preview-git-preview-pingcapdocs-cn16200-oreoxmt.vercel.app/zh/tidb/stable/dev-guide-sample-application-aws-lambda

File: develop/dev-guide-sample-application-nextjs.md
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:1]: Preview: https://pingcap-docsite-preview-git-preview-pingcapdocs-cn16200-oreoxmt.vercel.app/zh/tidb/stable/dev-guide-sample-application-nextjs

File: develop/dev-guide-sample-application-nodejs-sequelize.md
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nodejs-sequelize.md:1]: Preview: https://pingcap-docsite-preview-git-preview-pingcapdocs-cn16200-oreoxmt.vercel.app/zh/tidb/stable/dev-guide-sample-application-nodejs-sequelize

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
+---
+title: 在 AWS Lambda 函数中使用 mysql2 连接到 TiDB
+summary: 本文介绍如何在 AWS Lambda 函数中使用 TiDB 和 mysql2 构建一个 CRUD 应用程序，并给出了简单示例代码片段。
+---
+
+# 在 AWS Lambda 函数中使用 mysql2 连接到 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库。[AWS Lambda 函数](https://aws.amazon.com/lambda/) 是一个计算服务，[node-mysql2](https://github.com/sidorares/node-mysql2) 是一个与 [mysqljs/mysql](https://github.com/mysqljs/mysql) 兼容的面向 Node.js 的 MySQL 驱动。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-aws-lambda.md:8]:
```suggestion
TiDB 是一个兼容 MySQL 的数据库。[AWS Lambda 函数](https://aws.amazon.com/lambda/)是一个计算服务，[mysql2](https://github.com/sidorares/node-mysql2) 是当前流行的开源 Node.js Driver 之一。
```

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+本文档将展示如何在 AWS Lambda 函数中使用 TiDB 和 mysql2 来完成以下任务：
+
+- 配置你的环境。
+- 使用 node-mysql2 驱动连接到 TiDB 集群。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-aws-lambda.md:13]:
```suggestion
- 使用 mysql2 驱动连接到 TiDB 集群。
```

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+- 推荐使用[连接池](https://github.com/sidorares/node-mysql2#using-connection-pools)来管理数据库连接，以减少频繁建立和销毁连接所带来的性能开销。
+- 为了避免 SQL 注入的风险，推荐使用[预处理语句](https://github.com/sidorares/node-mysql2#using-prepared-statements)执行 SQL。
+- 在不涉及大量复杂 SQL 语句的场景下，推荐使用 ORM 框架（例如：[Sequelize](https://sequelize.org/)、[TypeORM](https://typeorm.io/) 或 [Prisma](https://www.prisma.io/)）来提升你的开发效率。
+- 如需为你的应用程序构建一个 RESTful API，建议[将 AWS Lambda 与 Amazon API Gateway 结合使用](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/services-apigateway.html)。
+- 关于使用 TiDB Serverless 和 AWS Lambda 设计高性能应用程序的更多信息，可以参考这篇[博客文章](https://aws.amazon.com/blogs/apn/designing-high-performance-applications-using-serverless-tidb-cloud-and-aws-lambda/)。
+
+## 下一步
+
+- 关于在 AWS Lambda 函数中使用 TiDB 的更多细节，可以参考 [`TiDB-Lambda-integration/aws-lambda-bookstore` 示例程序](https://github.com/pingcap/TiDB-Lambda-integration/blob/main/aws-lambda-bookstore/README.md)。你也可以使用 AWS API Gateway 为你的应用程序构建 RESTful API。
+- 关于 node-mysql2 的更多使用方法，可以参考 [node-mysql2 的 GitHub 仓库](https://github.com/sidorares/node-mysql2)。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-aws-lambda.md:348]:
```suggestion
- 关于 mysql2 的更多使用方法，可以参考 [mysql2 的官方文档](https://sidorares.github.io/node-mysql2/zh-CN/docs)。
```

File: develop/dev-guide-sample-application-nextjs.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+---
+
+# 在 Next.js 中使用 mysql2 连接到 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库，[mysql2](https://github.com/sidorares/node-mysql2) 是当前流行的开源 Node.js Driver 之一。
+
+本文档将展示如何在 Next.js 中使用 TiDB 和 mysql2 来完成以下任务：
+
+- 配置你的环境。
+- 使用 node-mysql2 驱动连接到 TiDB 集群。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:13]:
```suggestion
- 使用 mysql2 驱动连接到 TiDB 集群。
```

File: develop/dev-guide-sample-application-nextjs.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+## 注意事项
+
+- 推荐使用[连接池](https://github.com/sidorares/node-mysql2#using-connection-pools)来管理数据库连接，以减少频繁建立和销毁连接所带来的性能开销。
+- 为了避免 SQL 注入的风险，推荐使用[预处理语句](https://github.com/sidorares/node-mysql2#using-prepared-statements)执行 SQL。
+- 在不涉及大量复杂 SQL 语句的场景下，推荐使用 ORM 框架（例如：[Sequelize](https://sequelize.org/)、[TypeORM](https://typeorm.io/) 或 [Prisma](https://www.prisma.io/)）来提升你的开发效率。
+
+## 下一步
+
+- 关于使用 ORM 框架和 Next.js 构建复杂应用程序的更多细节，可以参考 [tidb-prisma-vercel-demo](https://github.com/pingcap/tidb-prisma-vercel-demo)。
+- 关于 node-mysql2 的更多使用方法，可以参考 [node-mysql2 的 GitHub 仓库](https://github.com/sidorares/node-mysql2)。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:266]:
```suggestion
- 关于 mysql2 的更多使用方法，可以参考 [mysql2 的官方文档](https://sidorares.github.io/node-mysql2/zh-CN/docs)。
```

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+    npm run build
+
+    # Bundle for AWS Lambda
+    # =====================
+    # dist/index.zip
+    ```
+
+2. 访问 [AWS Lambda 控制台](https://console.aws.amazon.com/lambda/home#/functions)。
+
+3. 按照[使用 Node.js 构建 Lambda 函数](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/lambda-nodejs.html)中的步骤创建一个Node.js Lambda 函数。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-aws-lambda.md:233]:
```suggestion
3. 按照[使用 Node.js 构建 Lambda 函数](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/lambda-nodejs.html)中的步骤创建一个 Node.js Lambda 函数。
```

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 7]
```diff
@@ -0,0 +1,355 @@
...
+
+更多信息参考[删除数据](/develop/dev-guide-delete-data.md)。
+
+## 注意事项
+
+- 推荐使用[连接池](https://github.com/sidorares/node-mysql2#using-connection-pools)来管理数据库连接，以减少频繁建立和销毁连接所带来的性能开销。
+- 为了避免 SQL 注入的风险，推荐使用[预处理语句](https://github.com/sidorares/node-mysql2#using-prepared-statements)执行 SQL。
+- 在不涉及大量复杂 SQL 语句的场景下，推荐使用 ORM 框架（例如：[Sequelize](https://sequelize.org/)、[TypeORM](https://typeorm.io/) 或 [Prisma](https://www.prisma.io/)）来提升你的开发效率。
+- 如需为你的应用程序构建一个 RESTful API，建议[将 AWS Lambda 与 Amazon API Gateway 结合使用](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/services-apigateway.html)。
+- 关于使用 TiDB Serverless 和 AWS Lambda 设计高性能应用程序的更多信息，可以参考这篇[博客文章](https://aws.amazon.com/blogs/apn/designing-high-performance-applications-using-serverless-tidb-cloud-and-aws-lambda/)。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-aws-lambda.md:343]:
```suggestion
- 关于使用 TiDB Serverless 和 AWS Lambda 设计高性能应用程序的更多信息，可以参考[这篇博客](https://aws.amazon.com/blogs/apn/designing-high-performance-applications-using-serverless-tidb-cloud-and-aws-lambda/)。
```

File: develop/dev-guide-sample-application-nextjs.md [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+
+</SimpleTab>
+
+### 第 4 步：运行代码并查看结果
+
+1. 运行以下命令，执行示例代码：
+
+   ```bash
+   npm run dev
+   ```
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:151]:
````suggestion
    ```bash
    npm run dev
    ```
````

File: develop/dev-guide-sample-application-nextjs.md [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+
+   ```json
+   {
+     "results": [
+       {
+         "Hello World": "Hello World"
+       }
+     ]
+   }
+   ```
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:167]:
````suggestion
    ```json
    {
      "results": [
        {
          "Hello World": "Hello World"
        }
      ]
    }
    ```
````

File: develop/dev-guide-sample-application-nextjs.md [hunk 10]
```diff
@@ -0,0 +1,272 @@
...
+
+### 第 4 步：运行代码并查看结果
+
+1. 运行以下命令，执行示例代码：
+
+   ```bash
+   npm run dev
+   ```
+
+2. 打开浏览器并访问 `http://localhost:3000`（请查看你的终端以获取实际的端口号，默认为`3000`）。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:153]:
```suggestion
2. 打开浏览器，在地址栏输入 `http://localhost:3000`，访问示例应用程序。请查看你的终端以获取实际的端口号，默认为 `3000`。
```

File: develop/dev-guide-sample-application-nextjs.md [hunk 11]
```diff
@@ -0,0 +1,272 @@
...
+
+3. 保存 `.env` 文件。
+
+</div>
+
+</SimpleTab>
+
+### 第 4 步：运行代码并查看结果
+
+1. 运行以下命令，执行示例代码：
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nextjs.md:147]:
```suggestion
1. 运行示例应用程序：
```

File: develop/dev-guide-sample-application-nodejs-sequelize.md [hunk 12]
```diff
@@ -0,0 +1,312 @@
...
+### 第 4 步：运行代码并查看结果
+
+运行以下命令，执行示例代码：
+
+```shell
+npm start
+```
+
+<details>
+<summary>**预期输出结果（部分）：**</summary>
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nodejs-sequelize.md:173]:
```suggestion
<summary>预期输出结果（部分）：/summary>
```
image

**Review by Oreoxmt — APPROVED** [review]

File: develop/dev-guide-sample-application-nodejs-sequelize.md [hunk 13]
```diff
@@ -0,0 +1,312 @@
...
+### 第 4 步：运行代码并查看结果
+
+运行以下命令，执行示例代码：
+
+```shell
+npm start
+```
+
+<details>
+<summary>预期输出结果（部分）：/summary>
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nodejs-sequelize.md:173]:
```suggestion
<summary>预期输出结果（部分）：</summary>
```

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 14]
```diff
@@ -0,0 +1,355 @@
...
+    ```yaml
+    Environment:
+      Variables:
+        TIDB_HOST: {tidb_server_host}
+        TIDB_PORT: 4000
+        TIDB_USER: {prefix}.root
+        TIDB_PASSWORD: {password}
+    ```
+
+4. 参考[使用短期凭证进行身份验证](https://docs.aws.amazon.com/zh_cn/cli/latest/userguide/cli-authentication-short-term.html)文档，设置 AWS 环境变量:
```
Comment (qiancai) [inline develop/dev-guide-sample-application-aws-lambda.md:176]:
```suggestion
4. 参考[使用短期凭证进行身份验证](https://docs.aws.amazon.com/zh_cn/cli/latest/userguide/cli-authentication-short-term.html)文档，设置 AWS 环境变量：
```

File: develop/dev-guide-sample-application-aws-lambda.md [hunk 15]
```diff
@@ -0,0 +1,355 @@
...
+
+3. 按照[使用 Node.js 构建 Lambda 函数](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/lambda-nodejs.html)中的步骤创建一个 Node.js Lambda 函数。
+
+4. 按照 [Lambda 部署程序包](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/gettingstarted-package.html#gettingstarted-package-zip)中的步骤，上传 `dist/index.zip` 文件。
+
+5. 在 Lambda 函数中[复制并配置相应的连接字符串](https://docs.aws.amazon.com/zh_cn/lambda/latest/dg/configuration-envvars.html)。
+
+    1. 在 Lambda 控制台的[**函数**](https://console.aws.amazon.com/lambda/home#/functions)页面中，选择**配置** > **环境变量**。
+    2. 点击**编辑**。
+    3. 为添加数据库访问凭证，按照以下步骤进行操作：
```
Comment (qiancai) [inline develop/dev-guide-sample-application-aws-lambda.md:241]:
```suggestion
    3. 按照以下步骤添加数据库访问凭证：
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**